app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+class CopyFileError(Exception):
+    pass
+
+
+class InvalidCommandError(CopyFileError):
+    pass
+
+
+class SameFileError(CopyFileError):
+    pass
+
+
+class CustomFileNotFoundError(CopyFileError):
+    pass
+
+
+class FileCopyError(CopyFileError):
+    pass
+
+
+def copy_file(copy_command: str) -> None:
+    command, source_file, destination_file = copy_command.split()
+
+    if command != "cp" or len(copy_command.split()) != 3:
+        raise InvalidCommandError("Invalid command format")
+    if source_file == destination_file:
+        raise SameFileError(f"{source_file} "
```

Comment:
As per task requirement:

> It must do nothing in case the user is trying to copy file to file with the same name.

So it would make sense to use a single if statement where you check if len is correct, if command is correct and if file name is not the same. That is much easier then making multiple ifs and raising exceptions in them